### ISSUE TITLE
CI unable to utilize rustc

### ISSUE THREAD

**Issue by aneksteind** [immunant/c2rust#427]
```
error[E0463]: can't find crate for `rustc_ast`
 --> dynamic_instrumentation/src/lib.rs:2:1
  |
2 | extern crate rustc_ast;
  | ^^^^^^^^^^^^^^^^^^^^^^^ can't find crate
  |
  = help: maybe you need to install the missing components with: `rustup component add rust-src rustc-dev llvm-tools-preview`
```

example build failure [here](
https://dev.azure.com/immunant/c2rust/_build/results?buildId=1651&view=logs&j=f7d03aff-a499-5cb4-81d5-3cd2462ec94c&t=85aa9f27-1ac1-5880-75f0-9dbc1339a40e)

The likely culprit seems to be `rust-src`

**Comment by fw-immunant** [comment]
From discussion last Friday, we need to rebuild our [CI docker images](https://hub.docker.com/r/immunant/c2rust/tags) from a commit at or after d1ba197fa677f78ed3be8f18c73f99555e889d3c. cc @thedataking.

**Comment by fw-immunant** [comment]
This has been resolved (CI passed for #418); see https://hub.docker.com/r/immunant/c2rust/tags for current image hashes.
